Source/Tools/FEXQonfig/main.qml

```diff
@@ -0,0 +1,669 @@
+                    text: qsTr("Multiblock")
+                    config: "Multiblock"
+                }
+
+                RowLayout {
+                    Layout.fillWidth: true
+
+                    Label { text: qsTr("Block size:") }
+                    ConfigSpinBox {
+                        config: "MaxInst"
+                        from: 0
+                        to: 1 << 30
+                    }
+                }
+
+                GroupBox {
+                    title: qsTr("JIT caches:")
+                    Layout.fillWidth: true
+
+                    ColumnLayout {
+                        anchors.left: parent ? parent.left : undefined
+                        anchors.right: parent ? parent.right : undefined
+
+                        ConfigCheckBox {
+                            text: qsTr("Generate AOT")
+                            config: "AOTIRGenerate"
+                        }
+                        ConfigCheckBox {
+                            text: qsTr("Capture AOT")
+                            config: "AOTIRCapture"
+                        }
+                        ConfigCheckBox {
+                            text: qsTr("Load AOT")
+                            config: "AOTIRLoad"
+                        }
+
+                        RowLayout {
+                            Label { text: qsTr("Cache object code:") }
+
+                            ButtonGroup {
+                                buttons: cacheObjCodeRadios.children
+
+                                checkedButton: ConfigModel.has("CacheObjectCodeCompilation", refreshCache)
+                                                ? cacheObjCodeRadios.children[ConfigModel.getInt("CacheObjectCodeCompilation", refreshCache)]
+                                                : null
+                            }
+
+                            RowLayout {
+                                id: cacheObjCodeRadios
+                                RadioButton {
+                                    text: qsTr("Off")
+                                }
+                                RadioButton {
+                                    text: qsTr("Read-only")
+                                }
+                                RadioButton {
+                                    text: qsTr("Read & write")
+                                }
+                            }
+                        }
+                    }
+                }
+
+                ConfigCheckBox {
+                    text: qsTr("Reduced x87 precision")
+                    config: "X87ReducedPrecision"
+                }
+
+                ConfigCheckBox {
+                    text: qsTr("Unsafe local flags optimization")
+                    config: "ABILocalFlags"
+                }
+
+                ConfigCheckBox {
+                    text: qsTr("Disable JIT optimization passes")
+                    config: "O0"
+                }
+            }
+        }
+
+        // Advanced settings
```

Comment:
It would also be nice to add a small help/description for them I think.

Reply:
Yup, though for now I'm just aiming to rebuild functionality that already exists (plus some things that were easy to add).

We already have descriptions written down in the Config.json file, but they're not readily available in C++ code yet, so this would need more work.